package/MDAnalysis/analysis/density.py

```diff
@@ -785,709 +822,3 @@ def __repr__(self):
-
-    Parameters
-    ----------
-    universe : MDAnalysis.Universe
-            :class:`MDAnalysis.Universe` object with a trajectory
-    select : str (optional)
-            selection string (MDAnalysis syntax) for the species to be analyzed
-            ["name OH2"]
-    delta : float (optional)
-            bin size for the density grid in Angstrom (same in x,y,z) [1.0]
-    start : int (optional)
-    stop : int (optional)
-    step : int (optional)
-            Slice the trajectory as ``trajectory[start:stop:step]``; default
-            is to read the whole trajectory.
-    metadata : dict. optional
-            `dict` of additional data to be saved with the object; the meta data
-            are passed through as they are.
-    padding : float (optional)
-            increase histogram dimensions by padding (on top of initial box size)
-            in Angstrom. Padding is ignored when setting a user defined grid. [2.0]
-    soluteselection : str (optional)
-            MDAnalysis selection for the solute, e.g. "protein" [``None``]
-    cutoff : float (optional)
-            With `cutoff`, select "<atomsel> NOT WITHIN <cutoff> OF <soluteselection>"
-            (Special routines that are faster than the standard ``AROUND`` selection);
-            any value that evaluates to ``False`` (such as the default 0) disables this
-            special selection.
-    update_selection : bool (optional)
-            Should the selection of atoms be updated for every step? [``False``]
-
-            - ``True``: atom selection is updated for each frame, can be slow
-            - ``False``: atoms are only selected at the beginning
-    verbose : bool (optional)
-            Print status update to the screen for every *interval* frame? [``True``]
-
-            - ``False``: no status updates when a new frame is processed
-            - ``True``: status update every frame (including number of atoms
-              processed, which is interesting with ``update_selection=True``)
-    interval : int (optional)
-           Show status update every `interval` frame [1]
-    parameters : dict (optional)
-            `dict` with some special parameters for :class:`Density` (see docs)
-    gridcenter : numpy ndarray, float32 (optional)
-            3 element numpy array detailing the x, y and z coordinates of the
-            center of a user defined grid box in Angstrom [``None``]
-    xdim : float (optional)
-            User defined x dimension box edge in Angstrom; ignored if
-            gridcenter is ``None``
-    ydim : float (optional)
-            User defined y dimension box edge in Angstrom; ignored if
-            gridcenter is ``None``
-    zdim : float (optional)
-            User defined z dimension box edge in Angstrom; ignored if
-            gridcenter is ``None``
-
-    Returns
-    -------
-    :class:`Density`
-            A :class:`Density` instance with the histogrammed data together
-            with associated metadata.
-
-
-    Notes
```

Comment:
There’s quite a bit of docs. Are these docs (at least in spirit) available in the module/DensityAnalysis?

(Sorry, a bit difficult to see this from the diff.)

Reply:
Looks to me like you moved everything that was necessary over when you created `DensityAnalysis` :)
The only thing I could see that was "missing" was a `Return` entry, so I added that (emphasizing on the fact that this is a physical density so users are aware).

Reply:
thanks for checking & updating, much appreciated!